llvm/test/CodeGen/NVPTX/st-param-imm.ll

```diff
@@ -0,0 +1,592 @@
+; RUN: llc < %s -march=nvptx64 | FileCheck %s
```

Comment:
This test begs to be autogenerated by llvm/utils/update_llc_test_checks.py

Reply:
Done.